README.rst

```diff
@@ -833,8 +833,9 @@ your loopback and RFC1918 IP addresses are blacklisted.
 
 This also requires the optional lxml and netaddr python dependencies to be
 installed.  This in turn requires the libxml2 library to be available - on
-Debian/Ubuntu this means ``apt-get install libxml2-dev``, or equivalent for
-your OS.
+Debian/Ubuntu this means ``apt-get install libxml2-dev && pip install lxml``,
```

Comment:
You need `lxml` whether or not you're on debian, as the first sentence says. It isn't anything to do with `libxml2`.

Reply:
And why did i get the error, that the package is missing eventually, only right after I enabled url-previews?

Reply:
yes, you need lxml for url-previews.

However, you have written:

"This ... requires the libxml2 library to be available - on Debian/Ubuntu this means ... ``pip install lxml``"

which is incorrect.